certificate-service/app/src/main/resources/fk7426/schematron/lu_tfp_asb_18.v1.sch

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<iso:schema
+  xmlns="http://purl.oclc.org/dsdl/schematron"
+  xmlns:iso="http://purl.oclc.org/dsdl/schematron"
+  queryBinding='xslt2'
+  schemaVersion='ISO19757-3'>
+
+  <iso:title>Schematron file for "Läkarutlåtande tillfällig föräldrapenning för ett allvarligt sjukt barn som inte har fyllt 18" - Version
+    1.
+  </iso:title>
+
+  <iso:ns prefix="xs" uri="http://www.w3.org/2001/XMLSchema"/>
+  <iso:ns prefix="rg"
+    uri="urn:riv:clinicalprocess:healthcond:certificate:RegisterCertificateResponder:3"/>
+  <iso:ns prefix="gn" uri="urn:riv:clinicalprocess:healthcond:certificate:3"/>
+  <iso:ns prefix="tp" uri="urn:riv:clinicalprocess:healthcond:certificate:types:3"/>
+
+
+  <iso:pattern id="intyg">
+    <iso:rule context="//rg:intyg">
+      <iso:assert test="count(gn:svar[@id='1']) ge 1">
+        Ett 'Läkarutlåtande tillfällig föräldrapenning för ett allvarligt sjukt barn som inte har fyllt 18' måsta ha ett 'Grund för
+        medicinskt underlag' svar
+      </iso:assert>
+      <iso:assert test="count(gn:svar[@id='1']) le 5">
+        Ett 'Läkarutlåtande tillfällig föräldrapenning för ett allvarligt sjukt barn som inte har fyllt 18' får ha max 5 'Grund för
+        medicinskt underlag'
+      </iso:assert>
+      <iso:assert test="count(gn:svar[@id='58']) = 1 or count(gn:svar[@id='55']) = 1">
+        Ett 'Läkarutlåtande tillfällig föräldrapenning för ett allvarligt sjukt barn som inte har fyllt 18' måste minst ha ett svar på
+        någon av frågorna 'Barnets diagnos' eller 'Fyll i vilka symtom barnet har om diagnos inte är
+        fastställd'
+      </iso:assert>
+      <iso:assert test="count(gn:svar[@id='59']) = 1">
+        Ett 'Läkarutlåtande tillfällig föräldrapenning för ett allvarligt sjukt barn som inte har fyllt 18' måste ha ett 'Hälsotillstånd'
```

Comment:
Maybe we should update ```Hälsotillstånd``` to ```Barnets hälsotillstånd``` to more precise indicate what question we are talking about